### PR TITLE
Auth/store oauth data

### PR DESCRIPTION
**Pull request by Cavonstavant**
**Create missing dummy data in prisma studio and test using the route `/test_oauth_data/:user_token`**

Steps:
1. Seed the db
2. Open prisma sutdio and
    a. Create a new  `OAuthProvider`
    b. Create a new `AuthUserData` and link it the the OAuthProvider and the User
3. Send a request to `/test_oauth_data/:user_token` where  `:user_token` is the tokenId in the Token table

> **Warning**
> After this PR you will need to drop your current db and apply a clean migration

### CODE REVIEW

File: database/src/config/seed.ts [hunk 1]
```diff
@@ -2,26 +2,52 @@ import { PrismaClient, Prisma } from '@prisma/client';
 import { env } from 'process';
 const prisma = new PrismaClient();
 
+const dummyOAuthProviderData : Prisma.oAuthProviderCreateInput[] = [
+    {
+        name: 'discord',
+    },
+    {
+        name: 'deams',
```
Comment (Bootoyka) [inline database/src/config/seed.ts:10]: deams ? Not sure about that

**Review by Bootoyka — COMMENTED** [review]
looks good, what about the little mistake and eslint rapport ?

File: database/src/config/seed.ts (hunk 1 above)
Comment (Cavonstavant) [reply database/src/config/seed.ts:10]: Id's fine really Ton'd worry aboud id

**Review by aurelenc — APPROVED** [review]
What @Bootoyka said + ok

**Pull request merged**